### PR TITLE
Python3 and Ansible-free

### PR DESCRIPTION
**Pull request by amerck**

### CODE REVIEW

**Comment by JesseBowling** [comment]
@amerck : The env file includes a number of quoted strings; shouldn't these be unquoted?

File: hpfeeds-logger/hpfeedslogger/processors.py [hunk 1]
```diff
@@ -111,7 +120,7 @@ def glastopf_event(identifier, payload):
     try:
         dec = ezdict(json.loads(str(payload)))
     except:
-        print 'exception processing glastopf event'
+        logger.warning('exception processing glastopf event')
         traceback.print_exc()
```
Comment (JesseBowling) [inline hpfeeds-logger/hpfeedslogger/processors.py:124]: in hpfeeds-cif you put this traceback into the logger stream; same treatment here?

Comment (JesseBowling) [reply hpfeeds-logger/hpfeedslogger/processors.py:124]: Several more of these later in the file

File: hpfeeds-logger/hpfeedslogger/processors.py [hunk 2]
```diff
@@ -247,7 +256,7 @@ def kippo_cowrie_sessions(identifier, payload, name, channel):
     try:
         dec = ezdict(json.loads(str(payload)))
     except:
-        print 'exception processing {} event'.format(name_lower)
+        logger.warning('exception processing %s event' % name_lower)
```
Comment (JesseBowling) [inline hpfeeds-logger/hpfeedslogger/processors.py:259]: I prefer `'string {0}'.format(var)` format. I can PR it later ;)

File: hpfeeds-logger/hpfeedslogger/processors.py (hunk 1 above)
Comment (amerck) [reply hpfeeds-logger/hpfeedslogger/processors.py:124]: Honestly, I put the traceback in the logger stream so I could see it while testing (it was throwing an error, but not actually displaying the error message). Honestly, we should probably just assign the error to a variable in these, and log those errors.

**Review by JesseBowling — APPROVED** [review]
LGTM!

**Pull request merged**